src/transforms/GetExternalFunctions.cpp

```diff
@@ -0,0 +1,117 @@
+/**
```

Comment:
general comment: add meaningful comments instead of copy-and-paste from other passes

Reply:
Sorry about that, I’ll do that and update soon